catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/HiveCatalogOperations.java

```diff
@@ -284,7 +284,9 @@ public HiveSchema alterSchema(NameIdentifier ident, SchemaChange... changes)
           properties.keySet());
 
       for (SchemaChange change : changes) {
-        if (change instanceof SchemaChange.SetProperty) {
+        if (change instanceof SchemaChange.UpdateComment) {
+          properties.put(COMMENT, ((SchemaChange.UpdateComment) change).getNewComment());
```

Comment:
You should use `database.setDescription` to change the Hive database's comment, instead of update the property